packages/wouter/test/link.test-d.tsx

```diff
@@ -1,44 +1,60 @@
-import { describe, it, assertType } from "vitest";
+import { describe, it } from "vitest";
 import { Link } from "wouter";
 import * as React from "react";
 
-describe("Link types", () => {
+describe("<Link /> types", () => {
   it("should have required prop href", () => {
     // @ts-expect-error
-    assertType(<Link>test</Link>);
-    assertType(<Link href="/">test</Link>);
+    <Link>test</Link>;
+    <Link href="/">test</Link>;
   });
 
-  it("should support state prop", () => {
-    assertType(
-      <Link href="/" state={{ a: "foo" }}>
-        test
-      </Link>
-    );
-    assertType(
-      <Link href="/" state={null}>
-        test
-      </Link>
-    );
-    assertType(
-      <Link href="/" state={undefined}>
-        test
-      </Link>
-    );
-    assertType(
-      <Link href="/" state="string">
-        test
-      </Link>
-    );
+  it("should throw error when `to` and `href` props are used in same time", () => {
```

Comment:
I would rephrase this, because technically it doesn't throw an error. Maybe:
```suggestion
  it("does not allow `to` and `href` props to be used at the same time", () => {
```